components/provisioner/internal/operations/stages/provisioning/create_operators_bindings.go

```diff
@@ -72,11 +80,17 @@ func (s *CreateBindingsForOperatorsStep) TimeLimit() time.Duration {
 }
 
 func (s *CreateBindingsForOperatorsStep) Run(cluster model.Cluster, _ model.Operation, log logrus.FieldLogger) (operations.StageResult, error) {
-	if cluster.Kubeconfig == nil {
-		return operations.StageResult{}, operations.ErrKubeconfigNil
+
+	var kubeconfig []byte
+	{
+		var err error
+		kubeconfig, err = s.dynamicKubeconfigProvider.FetchFromRequest(cluster.ClusterConfig.Name)
+		if err != nil {
+			return operations.StageResult{}, err
```

Comment:
I think better idea is to retry the operation in the case when Fetching kubeconfing from Gardener fails 

```suggestion
			return operations.StageResult{Stage: s.Name(), Delay: 20 * time.Second}, nil
```

Reply:
Fixed by myslef